src/model/vehicleLocation.ts

```diff
@@ -0,0 +1,37 @@
+export interface VehicleLocation {
+  id: number
+  siri_snapshot_id: number
+  siri_ride_stop_id: number
+  recorded_at_time: string
+  lon: number
+  lat: number
+  bearing: number
+  velocity: number
+  distance_from_journey_start: number
+  distance_from_siri_ride_stop_meters: number
+  siri_snapshot__id: number
+  siri_snapshot__snapshot_id: string
+  siri_snapshot__etl_status: string
+  siri_snapshot__etl_start_time: string
+  siri_snapshot__etl_end_time: string
+  siri_snapshot__error: string
+  siri_snapshot__num_successful_parse_vehicle_locations: number
+  siri_snapshot__num_failed_parse_vehicle_locations: number
+  siri_snapshot__num_added_siri_rides: number
+  siri_snapshot__num_added_siri_ride_stops: number
+  siri_snapshot__num_added_siri_routes: number
+  siri_snapshot__num_added_siri_stops: number
+  siri_snapshot__last_heartbeat: string
+  siri_snapshot__created_by: string
+  siri_ride__id: number
+  siri_ride__journey_ref: string
+  siri_ride__scheduled_start_time: string
+  siri_ride__vehicle_ref: string
+  siri_ride__first_vehicle_location_id: number
+  siri_ride__last_vehicle_location_id: number
+  siri_ride__duration_minutes: number
+  siri_ride__gtfs_ride_id: number
+  siri_route__id: number
+  siri_route__line_ref: number
+  siri_route__operator_ref: number
+}
```

Comment:
use generated model instead of doing it yourself

Reply:
I don't know how to